js-plugins/vue/src/utils.ts

```diff
@@ -54,6 +56,72 @@ export function callWithErrorHandle<
   }
 }
 
+export function isArray(val: any) {
+  return Array.isArray(val);
+}
+
+export function isRegExp(reg: any) {
```

Comment:
Perhaps declaring functions in this way can provide better type guarding abilities.

```diff
- export function isRegExp(reg: any) {
+ export function isRegExp(reg: unknown): reg is RegExp {
```

Reply:
I'll fix it with a new pull request.